.pre-commit-config.yaml

```diff
@@ -1,4 +1,9 @@
 repos:
+  - repo: https://github.com/shellcheck-py/shellcheck-py
+    rev: v0.8.0.4
+    hooks:
+      - id: shellcheck
+        args: [--external-sources]
```

Comment:
Here on the other hand I see that it might be useful when a change in the one tool (but unchanged util) causes the pre-commit hook to only get the changed file as input and the util is considered an external source...